soscollector/sosnode.py

```diff
@@ -275,7 +275,7 @@ def is_installed(self, pkg):
     def run_command(self, cmd, timeout=180, get_pty=False, need_root=False):
         '''Runs a given cmd, either via the SSH session or locally'''
         if cmd.startswith('sosreport'):
-            cmd = cmd.replace('sosreport', '/usr/sbin/sosreport')
+            cmd = cmd.replace('sosreport', self.host.sos_path_strip)
```

Comment:
`sos_path_strip` is used for stripping any part of the final sosreport path when we go to collect it from the node. E.G. on Atomic Host, it is set to `/host` because we have to run sos in a container - in the container it is written to `/host/var/tmp`, which is the host's real `/var/tmp/`. When we go to collect it, we no longer use the container so we have to strip the top-level directory.

We should probably make this something like `sos_bin_path`.